src/interfaces/coral_web/src/hooks/agents.ts

```diff
@@ -47,10 +47,11 @@ export const useAgent = ({ agentId }: { agentId?: string }) => {
   const cohereClient = useCohereClient();
   return useQuery({
     queryKey: ['agent', agentId],
-    enabled: !!agentId,
     queryFn: async () => {
       try {
-        if (!agentId) throw new Error('Agent ID not found');
+        if (!agentId) {
+          return await cohereClient.getDefaultAgent();
```

Comment:
@tanzimCohere this method should return an `Agent` and it returns a "generic response" `{ message: 'ok' }`